<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidiFileImportDialog</class>
 <widget class="QWidget" name="MidiFileImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>224</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QGroupBox" name="midiFileGroupBox">
        <property name="title">
         <string>MIDI File</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <layout class="QHBoxLayout" name="midiFileLayout" stretch="0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="midiFileLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="midiFileOpenButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../mi_guiplugin/mi_guiplugin.qrc">
               <normaloff>:/mi_guiplugin/images/fileopen.png</normaloff>:/mi_guiplugin/images/fileopen.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="importSettingsGroupBox">
        <property name="title">
         <string>Import Settings</string>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QHBoxLayout" name="importTypeLayout" stretch="0,0,0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="importTypeLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Import Type</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="importTypeSpacer_1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="importTypeComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>96</width>
               <height>0</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
             <item>
              <property name="text">
               <string>Overwrite</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Append</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="importTypeSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>33</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="importBarlineLayout" stretch="0,0,0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="importBarlineLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Import Barlines</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="importBarlineSpacer_1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="importBarlineComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>96</width>
               <height>0</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
             <item>
              <property name="text">
               <string>Yes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>No</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="importBarlineSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>33</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../mi_guiplugin/mi_guiplugin.qrc"/>
 </resources>
 <connections/>
</ui>
